contracts/interchain-token-service/src/state.rs

```diff
@@ -281,6 +290,34 @@ pub fn save_token_config(
         .change_context(Error::Storage)
 }
 
+pub fn save_custom_token(
+    storage: &mut dyn Storage,
+    source_chain: ChainNameRaw,
+    register_token: RegisterToken,
```

Comment:
```suggestion
    RegisterToken { address, decimals, chain }: RegisterToken,
```
Destructure the arg maybe so the following code is cleaner

Reply:
wow i didn't know you could do this

Reply:
Haha, not commonly used, but it's neat sometimes